sw/device/tests/chip_power_sleep_load.c

```diff
@@ -358,6 +362,26 @@ bool test_main(void) {
   }
   LOG_INFO("AON Timer active");
 
+  // ADC Controller
+  const uint8_t kNumLowPowerSamples = 2;
+  const uint8_t kNumNormalPowerSamples = 1;
+  const uint8_t kPowerUpTimeAonCycles = 3;
+  const uint8_t kWakeUpTimeAonCycles = 100;
```

Comment:
@tjaychen does this configuration make sense to you?

Reply:
i think the number of power samples is just a test time thing.
For the powerup and wakeup time, please have a look [here](https://docs.opentitan.org/hw/top_earlgrey/ip/ast/doc/#adc)

It's probably best to use a function that converts from us to cycles, and have the constant here be the microsecond instead.

Reply:
You can use the `aon_timer_testutils_get_aon_cycles_from_us` function from the header file below to achieve that:
https://github.com/lowRISC/opentitan/blob/10d6e41577f6602010c342cf8575cd71f62029f6/sw/device/lib/testing/aon_timer_testutils.h#L12-L15

Reply:
Done.
Thank you for the review.

Reply:
Thanks for doing this. I did not however that the AST documentation defines a different wakeup time of `30 uS`. Is there a particular reason why that is the case?

Otherwise LGTM.

Reply:
Done